test/integration/clientHelper.ts

```diff
@@ -132,17 +157,11 @@ let _globalAdminUser: MatrixClient;
 async function getGlobalAdminUser(homeserver: string): Promise<MatrixClient> {
     // Initialize global admin user if needed.
     if (!_globalAdminUser) {
+        let accessToken: string;
         const USERNAME = "mjolnir-test-internal-admin-user";
-        try {
-            await registerUser(homeserver, USERNAME, USERNAME, USERNAME, true);
-        } catch (e) {
-            if (e.isAxiosError && e?.response?.data?.errcode === 'M_USER_IN_USE') {
-                // Then we've already registered the user in a previous run and that is ok.
-            } else {
-                throw e;
-            }
```

Comment:
For some historical context

This code is redundant and the version that "actually works" is [here](https://github.com/matrix-org/mjolnir/pull/528/files#diff-e4d8c2feb5651074e939c29456219343d3697d0a74e8476fd8d38121936d4089L57-L65). It's not flagged up by typescript as dead code because the error will be implicitly typed as `any` rather than `unknown`. 

`e.isAxiosError` is there because at one point axios was used, and then removed in order to just reuse the http client the matrix-bot-sdk brings (accessed via `getRequestFn`).